docs/release_notes/0.57.0.md

```diff
@@ -0,0 +1,20 @@
+# Release 0.57.0
+
+## Improvements
+
+- Make EKS 1.20 the default (#3943)
+- Add stack status UpdateComplete as valid scenario (#3948)
+
+## Bug Fixes
+
+- Fix panic for `get nodegroup` (#3960)
+- Fix --wait issue for eksctl upgrade nodegroup command (#3956)
+- Fix Config file schema docs page horizontal scroll (#3958)
+- Cloudformation stack validation error because of underscore in EKS cluster name (#3942)
```

Comment:
the name of this is not great, i think it should be:

"Validate cluster and nodegroup names to avoid cloudformation error" or something

Reply:
thanks, updated the PR title and the release notes